Geometry_Engine/Create/ShapeProfile.cs

```diff
@@ -39,6 +40,16 @@ public static partial class Create
 
         public static ISectionProfile ISectionProfile(double height, double width, double webthickness, double flangeThickness, double rootRadius, double toeRadius)
         {
+            if (height < flangeThickness * 2 + rootRadius * 2 || width < webthickness + rootRadius * 2 +toeRadius*2 || toeRadius > flangeThickness)
+            {
+                Engine.Reflection.Compute.RecordError("The ratio between inputs makes section inconceivable");
```

Comment:
If possible, would be good to specify in the error message which parameters that were involved in causing the fail. for example "The ratio between height, flangethickness and rootradius makes section inconceivable" if the first of the check here is the one causing problems.

Will make it easier for users to know why the section was failing.

Could also set up a small private helper method for returning the error for this, so you only have to specify which paramters are failing for the input. Let me know if you want to have a quick chat about this.